roles/openshift_repos/tasks/main.yaml

```diff
@@ -10,6 +10,11 @@
   - name: Ensure libselinux-python is installed
     package: name=libselinux-python state=present
 
+  - name: Remove openshift_additional.repo file
+    file:
+      dest: /etc/yum.repos.d/openshift_additional.repo
+      state: absent
```

Comment:
`when: openshift_additional_repos | default([]) | length == 0`  ?

Reply:
Cluster is created prior to [this change](https://github.com/openshift/openshift-ansible/commit/3b3f160130f1465a3c40b6d9f9cf66c5de6f9ad6#diff-48463786d2e8ab4f2fdc81c1371114c3) with `openshift_additional_repos` set:

```
openshift_additional_repos=[{'id': 'openshift-prod', 'name': 'openshift-prod',...}]
```
Running an upgrade or re-running install would leave `/etc/yum.repos.d/openshift_additional.repo` (because we no longer manage the file) and create `/etc/yum.repos.d/openshift-prod.repo`.

```
/etc/yum.repos.d/openshift-prod.repo
/etc/yum.repos.d/openshift_additional.repo (contains openshift-prod)
```
We'd then see something like this when we're setting openshift-version.

```
Repository openshift-prod is listed more than once in the configuration
```

Reply:
Yup, sorry I missed that this had all been refactored.